pkg/lspext/lspext.go

```diff
@@ -0,0 +1,35 @@
+package lspext
```

Comment:
`lspx` doesn't make sense, IMO. We should clearly communicate what it is (extensions).